fuel/app/classes/materia/widget/installer.php

```diff
@@ -540,6 +540,7 @@ public static function generate_install_params(array $manifest_data, string $pac
 			'flash_version'       => $manifest_data['files']['flash_version'],
 			'height'              => $manifest_data['general']['height'],
 			'width'               => $manifest_data['general']['width'],
+			'restrict_publish'    => Util_Validator::cast_to_bool_enum(isset($manifest_data['general']['restrict_publish']) ? $manifest_data['general']['restrict_publish'] : false),
```

Comment:
change to make sure the values being sent are enum column compatible  `'0', '1'` strings.

```suggestion
			'restrict_publish'    => isset($manifest_data['general']['restrict_publish']) ? Util_Validator::cast_to_bool_enum($manifest_data['general']['restrict_publish']) : '0'),
```